README.md

```diff
@@ -283,83 +283,68 @@ $credentials = Twitter::getCredentials([
 
 Sign in with twitter
 ```php
-Route::get('twitter/login', ['as' => 'twitter.login', function(){
-	// your SIGN IN WITH TWITTER  button should point to this route
-	$sign_in_twitter = true;
-	$force_login = false;
+use Atymic\Twitter\Facades\Twitter;
 
-	// Make sure we make this request w/o tokens, overwrite the default values in case of login.
-	Twitter::reconfig(['token' => '', 'secret' => '']);
-	$token = Twitter::getRequestToken(route('twitter.callback'));
+Route::get('twitter/login', ['as' => 'twitter.login', static function () {
+    // Make sure we make this request w/o tokens, overwrite the default values in case of login.
+    $twitter = Twitter::usingCredentials('', '');
```

Comment:
This feels a bit hacky, do you think it's possible to avoid this?

Reply:
There's definitely a way to avoid this;

For some reason I was sticking as close to the old method as possible with this PR to be honest. I just wanted it to work. Improvements will come for sure.